graphdatascience/session/aurads_sessions.py

```diff
@@ -67,15 +68,25 @@ def get_or_create(
         if err := wait_result.error:
             raise RuntimeError(f"Failed to create session `{session_name}`: {err}")
 
-        gds_url = wait_result.connection_url
+        gds_connection = DbmsConnectionInfo(
+            uri=wait_result.connection_url,
+            username=AuraDsSessions.GDS_SESSION_USER,
+            # hashing the password to avoid storing the actual db password in Aura
+            password=hashlib.md5(db_connection.password.encode()).hexdigest(),
+        )
 
         if not existing_session:
             gds_user = create_details.username
             self._change_initial_pw(
-                gds_url=gds_url, gds_user=gds_user, initial_pw=create_details.password, new_pw=db_connection.password
+                gds_url=gds_connection.uri,
+                gds_user=gds_user,
+                initial_pw=create_details.password,
+                new_pw=gds_connection.password,
             )
```

Comment:
for AuraDS sessions, its about parity with dedicated sessions.

however, this would break existing sessions - upgrading the client -> cannot connect anymore.
Maybe we should only introduce it for the dedicated sessions